packit_service/worker/events/comment.py

```diff
@@ -183,3 +190,125 @@ def get_dict(self, default_dict: Optional[dict] = None) -> dict:
         result["issue_id"] = self.issue_id
         result.pop("_issue_object")
         return result
+
+
+class CommitCommentEvent(AbstractCommentEvent):
+    _trigger: Union[GitBranchModel, ProjectReleaseModel] = None
+    _event: ProjectEventModel = None
+
+    def __init__(
+        self,
+        project_url: str,
+        comment: str,
+        comment_id: int,
+        commit_sha: str,
+        actor: str,
+        repo_name: str,
+        repo_namespace: str,
+    ) -> None:
+        super().__init__(
+            project_url=project_url,
+            comment=comment,
+            comment_id=comment_id,
+        )
+        self.repo_name = repo_name
+        self.repo_namespace = repo_namespace
+        self.actor = actor
+        self.commit_sha = commit_sha
+        self._tag_name: Optional[str] = None
+        self._git_ref: Optional[str] = None
+
+    @property
+    def tag_name(self) -> Optional[str]:
+        if not self._tag_name and "--release" in self.comment:
+            release_match = re.search(r"--release[\s=](?P<release>\S+)", self.comment)
+            if release_match:
+                self._tag_name = release_match.group("release")
+        return self._tag_name
+
+    @property
+    def git_ref(self) -> Optional[str]:
+        if not self._git_ref and "--release" not in self.comment:
```

Comment:
Just wondering if we don't need the `self._git_ref` attribute, somewhere in the code, when we already have the `self._tag_name` attribute. I would probably populate the `self._git_ref` when there is `--release` in the comment just to be safe. But if you think it is not necessary I am fine with it.

Reply:
good points! I will rather change it